File: 231206/두 개의 격자 비교하기/compare-two-grid.py
n,m = map(int,input().split())

arr1 = [list(map(int,input().split()))for _ in range(n)]
arr2 = [list(map(int,input().split()))for _ in range(n)]
result = [[1 for _ in range(m)] for _ in range(n)]

for row in range(n):
    for col in range(m):
        if arr1[row][col] == arr2[row][col]:
            result[row][col] = 0
        print(result[row][col],end=" ")
    print()

File: 240622/두 개의 격자 비교하기/compare-two-grid.js
const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split('\n');
let n = Number(input[0].split(" ")[0])
let m = Number(input[0].split(" ")[1])

let arr1 = input.slice(1,n+1).map((num) => num.split(' ').map(Number));
let arr2 = input.slice(n+1,n*2+1).map((row) => row.split(' ').map(Number));


for(let i=0;i<n;i++){
    ans = Array()
    for(let j=0;j<m;j++){
        if (arr1[i][j]==arr2[i][j]){
            ans.push(0)
        }
        else{
            ans.push(1)
        }
    }
    console.log(ans.join(" "))
}